_tools/release_docs/main.go

```diff
@@ -229,6 +230,18 @@ func rewriteCodeBlocks(fileBytes []byte) []byte {
 }
 
 var (
+	htmlPreviewRE = regexp.MustCompile(`https://htmlpreview.github.io/\?https://github.com/kubernetes/kubernetes/[^/]*`)
+)
+
+// For example, this rewrites
+// https://htmlpreview.github.io/?https://github.com/kubernetes/kubernetes/v1.0.2/docs/api-reference/definitions.html#_v1_pod
+// To
+// http://kubernetes.io/<outputDir>/docs/api-reference/definitions.html#_v1_pod
+func rewriteHTMLPreviewLinks(fileBytes []byte) []byte {
+	return htmlPreviewRE.ReplaceAll(fileBytes, []byte(fmt.Sprintf("http://kubernetes.io/%s", *outputDir)))
```

Comment:
also this `outputDir` needs to be changed as well, because the outputDir will be `_v1.1` now, while the link URL is still `v1.1`